react/src/components/PaymentDialog/PaymentDialog.stories.tsx

```diff
@@ -4,6 +4,11 @@ import React from 'react';
 import { ThemeName } from '../../themes';
 import PaymentDialog, { PaymentDialogProps } from './PaymentDialog';
 
+import { defaultArgs } from '../../../.storybook/schemas';
+import { getCurrencyTypeFromAddress } from '../../util/address';
+
+const defaultCurrency = getCurrencyTypeFromAddress(defaultArgs.to)
```

Comment:
This could actually be defined just as it is but exported in the schemas file to avoid the repetition.

Reply:
NVM would have to import from utils which is not clean anyways

Reply:
Nice catch!
I just improved the logic a little bit.